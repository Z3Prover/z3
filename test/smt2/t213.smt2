
; Copyright (c) 2015 Microsoft Corporation

(set-option :pp.decimal true)
(declare-const x Real)
(declare-const y Real)

(simplify (/ 1.0 (^ 2.0 (/ 1.0 2.0))))
(simplify (/ 3.0 (^ 2.0 (/ 1.0 2.0))))
(simplify (/ (^ 2.0 (/ 1.0 2.0)) 3.0))
(simplify (= (/ 1.0 (^ 2.0 (/ 1.0 2.0)))
             (/ (^ 2.0 (/ 1.0 2.0)) 2.0)))
(simplify (/ (^ 2.0 (/ 1.0 2.0)) (^ 3.0 (/ 1.0 2.0))))
(simplify (/ (^ 2.0 (/ 1.0 2.0)) 0.0))
(simplify (/ (^ 2.0 (/ 1.0 2.0)) 1.0))



