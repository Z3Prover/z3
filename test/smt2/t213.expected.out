0.7071067811?
2.1213203435?
0.4714045207?
true
0.8164965809?
(/ 1.4142135623? 0.0)
1.4142135623?
